jsonld-contexts/user.jsonld

```diff
@@ -26,6 +26,7 @@
         "isFirstConnection": "https://smartdatamodels.org/isFirstConnection",
         "isProfileEcogestureCompleted": "https://smartdatamodels.org/isProfileEcogestureCompleted",
         "isProfileTypeCompleted": "https://smartdatamodels.org/isProfileTypeCompleted",
+        "lastConnectionDate": "https://vocab.egm.io/lastConnectionDate",
```

Comment:
`lastLoginDate`?

Reply:
Works as well but since there's another prop called `isFirstConnection` I thought to keep it consistent